packages/core/lib/commands/debug/meta.js

```diff
@@ -59,6 +63,11 @@ module.exports = {
         option: "--compile-none",
         description:
           "Forces the debugger to use artifacts even if it detects a problem.  Dangerous; may cause errors."
+      },
+      {
+        option: "--url",
+        description:
+          "Connects to a specified provider given via URL, ignoring networks in config. NOTE: compileNone is set to true when a config file is not found AND url is specified"
```

Comment:
I think you can leave out the note; this is really a behind-the-scenes thing.  The `compileNone` option isn't dangerous in this context like it usually is.  Although hm... probably it should be made clearer that this allows using the debugger outside of a Truffle project.  I think that part is worth making explicit!  So maybe could just have a second sentence, "Allows using the debugger outside of a Truffle project."?

Reply:
Added there, let me know what you think